tracing/opentelemetry/pom.xml

```diff
@@ -27,6 +27,10 @@
     <artifactId>helidon-tracing-opentelemetry</artifactId>
     <name>Helidon Tracing Open Telemetry</name>
 
+    <properties>
+        <version.lib.opentelemetry-sdk-extension-autoconfigure>1.22.0-alpha</version.lib.opentelemetry-sdk-extension-autoconfigure>
```

Comment:
(I assume we're OK with this property at this level, and not in `dependencies/pom.xml` as would normally be the case.)

Reply:
My understanding is that for "tangential" dependencies we don't want to load up `dependencies/pom.xml` with them.

Now, maybe OTel is becoming central enough that we do want them there but ideally there should be direct dependencies on OTel only from one (maybe two?) artifacts.

Maybe that's just for integrations and this usage is less tangential than those. 

@tomas-langer @romain-grecourt Thoughts, folks?